Header header
float32[6] throttle		# motor trottle percentage of all 6 motors [0-1]